servers.json

```diff
@@ -794,6 +794,12 @@
         "name": "Beast MC",
         "addresses": [
             "beastmc.net"
+    },
```

Comment:
You must keep this line for the JSON validation checks to pass.

Reply:
Could you specify sorry I've misunderstood